README.md

```diff
@@ -7,7 +7,7 @@ Nautobot is a Network Source of Truth and Network Automation Platform.
 Nautobot runs as a web application atop the [Django](https://www.djangoproject.com/)
 Python framework with a [PostgreSQL](https://www.postgresql.org/) database. 
 
-The complete documentation for Nautobot can be found at [Read the Docs](https://nautobot.readthedocs.io/en/stable/).
+The complete documentation for Nautobot can be found at [Read the Docs](https://nautobot.readthedocs.io/).
```

Comment:
This should link to https://nautobot.readthedocs.io/en/latest/

Reply:
fixed.